k8s/helm/templates/ingress.yaml

```diff
@@ -8,18 +8,17 @@ metadata:
     helm.sh/chart: {{ include "chart.chart" . }}
     app.kubernetes.io/instance: {{ .Release.Name }}
     app.kubernetes.io/managed-by: {{ .Release.Service }}
-  {{- with .Values.ingress.annotations }}
   annotations:
-    {{- toYaml . | nindent 4 }}
-  {{- end }}
+      kubernetes.io/ingress.global-static-ip-name: survey-runner
+      {{- if .Values.ingress.managedCertificateName }}
+      ingress.gcp.kubernetes.io/pre-shared-cert: {{ quote .Values.ingress.managedCertificateName }}
+      {{- end }}
 spec:
   backend:
     serviceName: {{ .Chart.Name }}
     servicePort: 80
-{{- if .Values.ingress.tls }}
+{{- if .Values.ingress.tlsSecretName }}
   tls:
-  {{- range .Values.ingress.tls }}
-    - secretName: {{ .secretName }}
-  {{- end }}
+    - secretName: {{ .Values.ingress.tlsSecretName }}
```

Comment:
We could remove the `tlsSecretName` variable and just use `tls-cert` here:

```
{{- if not .Values.ingress.managedCertificateName }}
  tls:
    - secretName: tls-cert
{{- end }}
```

Reply:
Yep, that makes things simpler